Alicerce/Sources/Network/Mappable.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  Mappable.swift
+//  Alicerce
+//
+//  Created by Luís Afonso on 06/04/2017.
+//  Copyright © 2017 Mindera. All rights reserved.
+//
+
+import Foundation
+
+public enum MappableError: Error {
+
+    case custom(String)
+
+    /// Unwrapps the enum custom value
+    var description: String {
+        switch self {
+        case let .custom(description): return description
+        }
+    }
+}
+
+public protocol Mappable {
+
+    /// Converts the object from a `Any` (Dictionary) to Itself.
```

Comment:
the doc is wrong. it can be anything, i use to apply the `Mappable` to everything, not only for dictionaries and it can be an array, yes.
i will fix the doc.